src/stdlib_ascii.fypp

```diff
@@ -284,31 +291,58 @@ contains
 
     end function to_upper
 
-    !> Convert character variable to title case
+    !> Converts character sequence to title case
     !> ([Specification](../page/specs/stdlib_ascii.html#to_title))
     !>
     !> Version: experimental
     pure function to_title(string) result(title_string)
         character(len=*), intent(in) :: string
         character(len=len(string)) :: title_string
+        integer :: i
+        logical :: capitalize_switch
+
+        capitalize_switch = .true.
+        do i = 1, len(string)
+            if (is_alpha(string(i:i))) then
```

Comment:
I would prefer to use alphanumerical characters to define words.
```suggestion
            if (is_alphanum(string(i:i))) then
```

The result of ``to_title("1st, 2nd, 3rd")`` with `"1St, 2Nd, 3Rd"` seems surprising to me. Given that `to_sentence currently uses alphanumeric characters to determine the beginning of the first word.